node/impl/full/eth.go

```diff
@@ -1286,13 +1292,6 @@ func (e *EthEventHandler) EthGetLogs(ctx context.Context, filterSpec *ethtypes.E
 		if err != nil {
 			return nil, err
 		}
-
-		// should also have the minHeight in the filter indexed
-		if b, err := e.EventFilterManager.EventIndex.IsHeightProcessed(ctx, uint64(pf.minHeight)); err != nil {
-			return nil, xerrors.Errorf("failed to check if event index has events for the minHeight: %w", err)
-		} else if !b {
-			return nil, xerrors.Errorf("event index does not have event for epoch %d", pf.minHeight)
```

Comment:
This could just be a null block or an epoch we missed indexing because of en edge case where a node crashes before writing the events to the DB.

The right way to handle this is to introduce

a) Make event Index null block aware
b) Automated backfilling-auto repair in the event Index DB

Reply:
So the best we can do right now is to ensure that we are past the maxHeight in the range. It is not perfect but a strict improvement over what we have till we fix Indexing.

Reply:
agreed, but you should leave a note in here, at least a TODO, that this check should ideally be restored in some form

Reply:
Added docs.